dev/TabView/TabView.xaml

```diff
@@ -44,30 +44,35 @@
                                 Content="{TemplateBinding TabStripHeader}"
                                 ContentTemplate="{TemplateBinding TabStripHeaderTemplate}"/>
 
-                            <Grid x:Name="ShadowReceiver"
-                                Grid.ColumnSpan="4"/>
-
                             <primitives:TabViewListView
                                 Grid.Column="1"
                                 x:Name="TabListView"
+                                Padding="{TemplateBinding Padding}"
```

Comment:
@StephenLPeters @beervoley So should I close #3776 then?
If so, @beervoley can you please add "Closes #3736" to the PR description?

Reply:
@chingucoding you can check it in, I'll merge the master after :)

Reply:
I can't check in anything so I'll defer to @StephenLPeters or @ranjeshj on this :)

Reply:
I already merged in #3776, so we should be good as long as we merge with master cleanly.